File: C/PRAK403-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int main () {
    int awal, akhir, i, a;
    scanf("%d %d", &awal, &akhir);
    a=akhir;
    if (awal<akhir) {
        for (i=awal;i<=akhir;i++) {
            printf("%d ",i);
            if (a>awal) {
                printf("%d - ",a);}
            else {
                printf("%d ",a);}
            a=a-1;} 
    }
    else if (awal>akhir) {  
        for (i=awal;i>=akhir;i--) {
            printf("%d ", i);
            if (a<awal) {
                printf("%d - ",a);} 
            else {
                printf("%d ",a);}
            a=a+1;}
    }
}

File: PYTHON/PRAK403-2210817220029-SitiAinurRahmawati.py
awal, akhir = map(int, input().split())
a=akhir
if awal<akhir :
    for i in range (awal, akhir+1, 1) :
        print(i, end=" ");
        if a>awal :
            print(a, end=" - ")
        else :
            print(a)
        a=a-1 
elif awal>akhir :
    for i in range (awal, akhir-1, -1) :
        print(i, end=" ");
        if a<awal :
            print(a, end=" - ")
        else :
            print(a)
        a=a+1 